doc/release-notes-15936.md

```diff
@@ -0,0 +1,8 @@
+GUI changes
+-----------
+
+Configuration changes made in the bitcoin GUI (such as the pruning setting,
+proxy settings, UPNP preferences) are now saved to <datadir>/settings.json file
+rather than to the Qt settings backend (windows registry or unix desktop config
+files), and the GUI settings will now be used if bitcoind is started in
+subsequently, rather than ignored.
```

Comment:
This can be marked as resolved. I cannot do that for some reason even though I opened it.

_Is not the author of a suggestion the best one to judge whether it is resolved or not, github `~!(@*(*~!=&/_*`_